source/developer/developer-setup.md

```diff
@@ -37,11 +37,12 @@ Developer Machine Setup
 6. Run unit tests on Mattermost using `make test` to make sure the installation was successful
 7. If the tests passed, you need to create a team and an admin account. You can choose a team name, email and password:
 	- Note: Make sure your team name does not contain any spaces.
+    - Note: The latest version of mattermast does not rely on `godep` anymore. If you have the latest version, run the commands by omitting the `godep` at the beginning (e.g. `go run mattermost.go  -create_team -team_name="name" -email="user@example.com"`)
```

Comment:
Thanks for catching this @apheleia! Some minor tweaks:

1. I think we want to keep the docs up to date with the latest version of Mattermost. So I'd actually propose removing the `godep` part in each command and then add a note that `godep` is required for versions prior to vx.x

2. There is also a minor typo: "mattermast" -> "Mattermost"